src/style.css

```diff
@@ -0,0 +1,12 @@
+#filtergroup {
```

Comment:
Just emphasizing using classes instead of id's for styling:
```
.filtergroup {
...
}

.data {
...
```

Reply:
See comment above about using 'id' for DIVs.